blocks.go

```diff
@@ -21,6 +21,12 @@ func (srv *notesAPI) InsertBlock(ctx context.Context, req *notesv1.InsertBlockRe
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	// Check user is part of the group.
+	_, err = srv.groups.GetGroup(ctx, &models.OneGroupFilter{GroupID: req.GroupId}, token.AccountID)
+	if err != nil {
+		return nil, statusFromModelError(err)
+	}
+
```

Comment:
I think this is unnecessary for now because only the note author can edit the note so the authorisation check is already done in `InsertBlock`.